content/development/_index.html

```diff
@@ -13,7 +13,7 @@
 </ul>
 </div>
 
-<p class="intro">Welcome to the Development home for Haiku! Here is a general overview of the information available for developers who would like to contribute to the Haiku Project. To find out more about the developer community, check out the IRC and mailing list links in the box to the right.</p>
+<p class="intro">Welcome to the Development home for Haiku! Here is a general overview of the information available for developers who would like to contribute to the Haiku Project. To find out more about the developer community, check out the IRC and mailing list links given above in development community section.</p>
```

Comment:
I don't see much sense to have the "Development Community" title in line 9.
How about moving this paragraph above the links to IRC and MLs and end the paragraph with:

> ...check out these IRC and mailing list links: